Composer/packages/client/src/components/Notifications/NotificationPanel.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import React, { useCallback } from 'react';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { IPanelProps, Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import { IRenderFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { NeutralColors, SharedColors } from '@uifabric/fluent-theme';
+import { useRecoilValue } from 'recoil';
+import formatMessage from 'format-message';
+
+import { notificationsSelector } from '../../recoilModel/selectors/notificationsSelector';
+
+import { NotificationCard } from './NotificationCard';
+
+const styles = {
+  container: css`
+    display: flex;
+    position: absolute;
+    right: 0px;
+    top: 18px;
+  `,
+  empty: css`
+    color: ${NeutralColors.gray130};
+    height: 100%;
+    margin: 24px 0;
+    text-align: center;
+    width: 100%;
+  `,
+  panelButtons: css`
+    display: flex;
+    justify-content: center;
+  `,
+};
+
+interface NotificationPanelProps {
+  isOpen: boolean;
+  onDismiss: (event?: React.SyntheticEvent<HTMLElement, Event>) => void;
+  onDeleteNotification: (id: string) => void;
+}
+
+const NotificationPanel: React.FC<NotificationPanelProps> = ({ isOpen, onDeleteNotification, onDismiss }) => {
+  const notifications = useRecoilValue(notificationsSelector);
```

Comment:
NotificationPanel is half-smart, it knows how to get a list of notifications but don't know how to delete them. I think it should be fully smart, meaning remove the callback for delete and get the dispatcher directly.

OR

go fully "dumb component" and pass a list of notifications + delete callback.